osu.Game/Overlays/Options/OptionsFooter.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Modes;
+using OpenTK;
+using OpenTK.Graphics;
+
+namespace osu.Game.Overlays.Options
+{
+    public class OptionsFooter : FillFlowContainer
+    {
+        [BackgroundDependencyLoader]
+        private void load(OsuGameBase game, OsuColour colours)
+        {
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+            Direction = FillDirection.Vertical;
+            Padding = new MarginPadding { Top = 20, Bottom = 30 };
+
+            var modes = new List<Drawable>();
+
+            foreach (PlayMode m in Enum.GetValues(typeof(PlayMode)))

```

Comment:
nah. if we're going to do this here then it should be enforced solution-wide. not int he scope of this PR